includes/admin/welcome.php

```diff
@@ -104,20 +104,91 @@ public function remove_submenu_pages()
 	public function main_nav_links()
```

Comment:
This was a good refactoring opportunity. I would recommend in future to have small files, it gets easier to review.
You could have moved functions like this to a new file and included that file in this file.

Not needed to do in this PR, but it will be handy in future.

Reply:
okay